src/components/button/theme.ts

```diff
@@ -0,0 +1,55 @@
+import { SystemStyleObject } from '@chakra-ui/theme-tools';
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+import { sizes as buttonHeights, typography } from '../../theme/shared';
+
+const baseStyle: SystemStyleObject = {
+  borderRadius: 'sm',
+};
+
+const sizes = {
```

Comment:
I would invert the renaming, but perhaps that's just me so importing `sizes` and name the const something like `buttonSize` as you are creating the specific sizes her